packages/hash/frontend/src/pages/[account-slug]/types/entity-type/entity-type-tabs.tsx

```diff
@@ -1,28 +1,77 @@
-import { EntityType } from "@blockprotocol/type-system-web";
+import { extractBaseUri, VersionedUri } from "@blockprotocol/type-system-web";
 import { faPlus } from "@fortawesome/free-solid-svg-icons";
 import { FontAwesomeIcon } from "@hashintel/hash-design-system";
-import { Box, Typography } from "@mui/material";
+import { frontendUrl } from "@hashintel/hash-shared/environment";
+import { Box, Tabs, tabsClasses } from "@mui/material";
+import { useRouter } from "next/router";
+import { useEffect, useMemo, useState } from "react";
 import { EntityTypeDefinitionTab } from "./entity-type-definition-tab";
+import { EntityTypeEntitiesTab } from "./entity-type-entities-tab";
 import { TabButton } from "./tab-button";
+import { useEntityType } from "./use-entity-type";
+
+export const EntityTypeTabs = () => {
+  const router = useRouter();
+  const entityType = useEntityType();
+
+  const [activeTab, setActiveTab] = useState<string | undefined>(undefined);
+
+  useEffect(() => {
+    setActiveTab(`${frontendUrl}${router.asPath}`);
+  }, [router.asPath]);
+
+  const baseUri = useMemo(() => {
+    if (entityType?.$id) {
+      const entityTypeBaseUri = extractBaseUri(entityType.$id as VersionedUri);
+      return entityTypeBaseUri.substring(0, entityTypeBaseUri.length - 1);
```

Comment:
What is this for? If it's to remove a trailing slash, I'd use a RegExp to make it clearer what its intention is, e.g.
```suggestion
      return entityTypeBaseUri.replace(/\/$/, "");
```